src/language-graphql/printer-graphql.js

```diff
@@ -572,7 +574,11 @@ function printInterfaces(path, options, print) {
   return parts;
 }
 
-function clean(/*node, newNode , parent*/) {}
+function clean(node, newNode /* , parent */) {
+  if (node.kind === "StringValue" ) {
```

Comment:
```suggestion
    if (node.kind === "StringValue" && node.block && !node.value.includes("\n")) {
```

Reply:
Included the suggestion with comments. (Makes more sense indeed)